### ISSUE TITLE
Loading data error while executing cvppp_preproc_save.sh

### ISSUE THREAD

**Issue by anhtuanhsgs** [visinf/acis#5]
Hi, I have finished the procedure of reproducing your work to the step of executing `./runs/cvppp_preproc.sh`. But when I run `./runs/cvppp_pretrain.sh`, I got this error.

> Setting seed 130
> 
> /home/abc/torch/install/bin/lua: /home/abc/torch/install/share/lua/5.2/threads/threads.lua:183: [thread 2 callback] ./datasets/cvppp.lua:136: bad argument #1 to 'size' (dimension 1 out of range of 0D tensor at /home/abc/torch/pkg/torch/generic/Tensor.c:19)
> stack traceback:
> 	[C]: in function 'size'
> 	./datasets/cvppp.lua:136: in function <./datasets/cvppp.lua:135>
> 	(...tail calls...)
> 	[C]: in function 'xpcall'
> 	/home/abc/torch/install/share/lua/5.2/threads/threads.lua:234: in function 'callback'
> 	/home/abc/torch/install/share/lua/5.2/threads/queue.lua:65: in function </home/abc/torch/install/share/lua/5.2/threads/queue.lua:41>
> 	[C]: in function 'pcall'
> 	/home/abc/torch/install/share/lua/5.2/threads/queue.lua:40: in function 'dojob'
> 	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
> stack traceback:
> 	[C]: in function 'error'
> 	/home/abc/torch/install/share/lua/5.2/threads/threads.lua:183: in function 'dojob'
> 	/home/abc/torch/install/share/lua/5.2/threads/threads.lua:264: in function 'synchronize'
> 	/home/abc/install/share/lua/5.2/threads/threads.lua:142: in function 'specific'
> 	/home/abc/torch/install/share/lua/5.2/threads/threads.lua:125: in function </home/abc/torch/install/share/lua/5.2/threads/threads.lua:36>
> 	(...tail calls...)
> 	./dataloader.lua:54: in function '__init'
> 	/home/abc/torch/install/share/lua/5.2/torch/init.lua:91: in function </home/abc/torch/install/share/lua/5.2/torch/init.lua:87>
> 	[C]: in function 'DataLoader'
> 	./dataloader.lua:25: in function 'create'
> 	pretrain_main.lua:118: in main chunk
> 	[C]: in function 'dofile'
> 	...abc/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
> 	[C]: in ?

Can you please show me how to fix this?

**Comment by Khoa-NT** [comment]
#4 Same problem
